code/modules/SCP/SCPs/SCP-513.dm

```diff
@@ -28,15 +28,15 @@
 /obj/item/scp513/pickup(mob/living/user)
 	. = ..()
 	if(user.a_intent == I_HURT)
-		to_chat(user, "<span class='danger'><b><i>You accidentaly ring \the [src]!</i></b></span>")
+		to_chat(user, "<span class='danger'><b><i>You accidentally ring \the [src]!</i></b></span>")
 		ring(user)
 
 /obj/item/scp513/attack_self(mob/living/user)
 	ring(user)
 
 /obj/item/scp513/Process()
 	for(var/mob/living/carbon/M in victims)
-		if(prob(0.5))
+		if(prob(2.5))
```

Comment:
Prob 2.5, that's a probability of 2.5% out of 100, isn't it?

Reply:
Yep

Reply:
Explain the probability, and what it's meant to do if you could. Once I have a grasp on that function, then I can give you a pass or suggestion to fix.